src/components/routes/Host/ListingForm/ListingForm.container.ts

```diff
@@ -1,19 +1,18 @@
 import styled from 'styled-components';
 import { color } from 'styled/utils';
 
-const ListingFormContainer = styled.section`
+const ListingFormMobileContainer = styled.section`
   width: 100%;
 
-
   & > .bee-general-wrapper {
-    min-height: calc(100% - 64px);
     margin: 0 auto;
+    margin-bottom: 56px;
+    min-height: calc(100% - 64px);
+    min-width: auto;
```

Comment:
i dont think `auto` is a valid value for min-width, i think you're looking for `0`, but i also think you don;'t need to add it if it's not being set anywhere else?